sdks/typescript/examples/example.ts

```
@@ -0,0 +1,109 @@
+  for await (const session of _sessions) {
+    console.log('Session:', session.id);
+  }
+
+  console.log('Adding messages to session...');
+  await mySession.addMessages([
+    assistant.message('what did you have for breakfast today, alice?'),
+    alice.message('i had oatmeal.'),
+  ]);
+  console.log('Messages added.');
+
+  let sessionMetadata = await mySession.getMetadata();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'sessionMetadata' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)